### PR TITLE
Fix javascript linting - update dependencies to fix JS vulnerability (js-yaml)

### PR DESCRIPTION
**Pull request by lneves12**
## What does this PR change?

Fix javascript linting - update dependencies to fix vulnerability

https://github.com/davidtheclark/cosmiconfig/issues/180

## GUI diff

No difference.

## Documentation
- No documentation needed

- [ ] **DONE**

## Test coverage
- No tests

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"  
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"

### CODE REVIEW

File: susemanager-frontend/yarn.lock [hunk 1]
```diff
@@ -3598,14 +3597,22 @@ js-levenshtein@^1.1.3:
...
-js-yaml@^3.12.0, js-yaml@^3.9.0:
+js-yaml@^3.12.0:
   version "3.12.1"
   resolved "https://registry.yarnpkg.com/js-yaml/-/js-yaml-3.12.1.tgz#295c8632a18a23e054cf5c9d3cecafe678167600"
   integrity sha512-um46hB9wNOKlwkHgiuyEVAybXBjwFUV0Z/RaHJblRd9DXltue9FTYvzCr9ErQrK9Adz5MU4gHWVaNUfdmrC8qA==
   dependencies:
     argparse "^1.0.7"
     esprima "^4.0.0"
 
+js-yaml@^3.13.0:
```
Comment (ncounter) [inline susemanager-frontend/yarn.lock]: Do you see any reason why we have both `3.12.0` and `3.13.0` version? Who is requiring them? I see before it was even worse we with three version, but I wonder if we could go with one (the latest hopefully) only. What do you think?

Comment (lneves12) [reply susemanager-frontend/yarn.lock]: It's coming from eslint, yarn auditing is only auditing the dependencies.

**Comment by lneves12** [comment]
> Looks good. Changelogs are strictly not required in this case, as we already have them for other stuff, and FMPOV a changelog would not add anything useful for customers/users.
> 
> @lneves12, what's your opinion?

I don't think it's needed. Just added the check for it.

All the tests should be green now.

**Comment by lneves12** [comment]
I think this it’s good to be merged @juliogonzalez

**Pull request merged**